src/dscope.d

```diff
@@ -569,14 +569,30 @@ struct Scope
      */
     extern (C++) static void deprecation10378(Loc loc, Dsymbol sold, Dsymbol snew)
     {
+        // Bugzilla 15857
```

Comment:
> this is how we referenced Bugzilla in the rest of the code base

I know, but I have submitted PRs to start changing that, particulary for object.d.

>  this is a nitpick, so let's please move on with the beta, unless there is something else

Good point, but if you wind up amending this PR anyway, please amend that line.